backend/LeetWars.Core/LeetWars.Core.DAL/Context/SeedSettings/SeedDefaults.cs

```diff
@@ -26,6 +26,8 @@ public static class SeedDefaults
 
         public static readonly int BadgeSeed = 1337;
 
+        public static readonly int Friendship = 0228;
```

Comment:
why do we have different sets for different entities?

Reply:
idk, i saw that every seed id different